client/my-sites/plugins/education-footer/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { useLocalizeUrl } from '@automattic/i18n-utils';
+// import { useLocalizeUrl } from '@automattic/i18n-utils';
```

Comment:
Will need to restore this.

Reply:
Do the plugin routes need a locale attached?